developer.md

```diff
@@ -0,0 +1,89 @@
+# Developer startup
+
+Install dependencies:
+- Postgres
+- Node + npm
+- Python and Conda
+
+
+1. Clone repository:
+
+   ```shell
+   git clone git@github.com:populationgenomics/seqr.git
+   git checkout dev
+   ```
+
+1. Build UI
+
+   ```shell
+   cd ui/
+   npm install
+   npm run build
+   cp dist/* ../static/
+   cd ..
+   ```
+
+1. Build seqr python environment
+
+   ```shell
+   conda env create -n seqr python=3.7
```

Comment:
Apparently `conda env create` no longer works and it should `conda create`. 

https://github.com/conda/conda/issues/3859

Reply:
Thanks @daniaki!